client/my-sites/invites/controller.js

```diff
@@ -43,22 +42,18 @@ export function acceptInvite( context, next ) {
 			user().set( { email_verified: true } );
 		}
 		store.remove( 'invite_accepted' );
-		const acceptInviteCallback = ( error ) => {
-			if ( error ) {
-				debug( 'Accept invite error: ' + JSON.stringify( error ) );
-				page( window.location.href );
-			} else if ( get( acceptedInvite, 'site.is_vip' ) ) {
-				debug( 'Accepted invite for VIP sites' );
-				window.location.href = getRedirectAfterAccept( acceptedInvite );
```

Comment:
I see we're now changing to do a `page()` instead of `window.location.href` for VIP sites. Any chance this could subtly break something?

Reply:
`page( externalUrl )` should end up calling simply `window.location.href = externalUrl`. This is ensured at two places:

The `isOutsideCalypso` check here (probably redundant):

https://github.com/Automattic/wp-calypso/blob/trunk/client/boot/common.js#L102-L105

and also in `page.js` itself: if a route is unhandled, it will navigate with `window.location.href = path`:

https://github.com/visionmedia/page.js/blob/master/index.js#L649